docker/linux/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM philcryer/min-jessie:latest
```

Comment:
if you would ubuntu:zesty like I just did you would not have to install extra cmake but correct cmake is available in ubuntu:zesty.

```
docker pull ubuntu:zesty
docker run -ti ubuntu:zesty /bin/bash
apt-get update && apt-get upgrade -y
apt-get install git vim python cmake gcc g++ default-jdk maven
mkdir build
cd build
git clone https://github.com/drywolf/J2V8
export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64"
python prepare_build.py
python build.py -t linux -a x64
```